tests/support/setup.js

```diff
@@ -30,11 +31,11 @@ define([
         );
       } else {
         type = 'mouse' + shortType;
-        e = document.createEvent('MouseEvent');
-        e.initMouseEvent(
-          type, true, true, null, null, 0, 0, 0, 0, false, false,
-          false, false, 0, null
-        );
+        e = new MouseEvent(type, {
+          bubbles: true,
+          cancelable: true,
+          buttons: buttons
+        });
```

Comment:
The sane API should have given it away. :smiling_imp:  I'm having the same problem when spitballing jQuery 4.0 event work.